libs/cart/driver/magento/src/transforms/outputs/cart-item/simple-cart-item-transformer.ts

```diff
@@ -18,6 +19,7 @@ export function transformMagentoSimpleCartItem(cartItem: MagentoCartItem): DaffC
 		name: cartItem.product.name,
 		qty: cartItem.quantity,
 		price: cartItem.prices.price.value,
+    discount: daffDivide(cartItem.prices.total_item_discount.value, cartItem.quantity),
```

Comment:
I'm not sure that the math here is sound. What happens when there's two qty and an odd decimal. It appears we'd be off by a penny.

Reply:
I don't think that can happen, unless there's a problem with the backend. Presumably, the total discount on the backend is derived from the (discount of one item) * quantity. So the total discount will always be divisible by the quantity, unless the backend is giving us the wrong number of significant figures.

If the backend instead multiplies the price by the quantity, then multiplies by some percentage to get the total discount, then that seems like a magento/shopify bug. Though I doubt this is how it works, because it doesn't really make sense.